src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

```diff
@@ -69,6 +69,8 @@ object KafkaConfig {
 object SensorsConfig {
   def getThreadPoolSize: Int =
     Configs.conf.getInt("scheduler.sensors.thread.pool.size")
+  def getChangedSensorsMinimalChunkQuerySize: Int =
```

Comment:
As in the other PR, I think we should use `val` instead of `def` to fail fast

Reply:
ok